docs/tutorials/ruby-driver-create-client.txt

```diff
@@ -740,18 +740,24 @@ URI options are explained in detail in the :manual:`Connection URI reference
    * - compressors=Strings
      - ``:compressors => Array<String>``
 
-       Specified as a comma-separated list. The Ruby driver currently supports snappy
+       Specified as a comma-separated list. The Ruby driver currently supports snappy, zstd
```

Comment:
I suggest placing zstd ahead of snappy since my understanding is it is a better choice (better compression at the same speed).

Reply:
It seems this section is missing the explicit note that the driver tries to use the compression algorithms  in the order they are specified. I suggest replacing the first sentence with

> Specified as an ordered, comma-separated list.